.coderabbit.yaml

```
@@ -0,0 +1,70 @@
+  poem: true
+  
+  # Path filters and instructions
+  path_filters: []  # Add glob patterns to include/exclude files
+  path_instructions: []  # Add path-specific review instructions
+  
+  # Auto review settings
+  auto_review:
+    enabled: true
+    auto_incremental_review: true
+    drafts: false
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)